### PR TITLE
Easyconfigs for wheel using the foss 2016a toolchain.

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
Easyconfigs for installing the python wheel module.

### CODE REVIEW

File: easybuild/easyconfigs/w/wheel/wheel-0.29.0-foss-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/w/wheel/wheel-0.29.0-foss-2016a-Python-2.7.11.eb]: @FokkeDijkstra I would include a custom `sanity_check_paths` in here (& the other `wheel` easyconfig):

```python
sanity_check_paths = {
    'files': ['bin/wheel'],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}
```

**Comment by boegel** [comment]
(close/open to trigger Travis, it missed this PR for some reason...)

**Comment by FokkeDijkstra** [comment]
@Boegel. Thanks for the suggestion. sanity_check_paths have been added.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9204c2e217c231c8e3db03012fa3380d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2681.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cb872feaf17ede18d675dcf0c3a30e6c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @FokkeDijkstra!

**Pull request merged**